Tasks/AzureCloudPowerShellDeployment/Publish-AzureCloudDeployment.ps1

```diff
@@ -156,36 +159,34 @@ function Get-DiagnosticsExtensions($storageAccount, $extensionsPath)
         }
         else
         {
-            Write-Warning "Could not get the primary storage key for storage account '$storageAccount'.  Unable to apply any diagnostics extensions."
+            Write-Warning (Get-LocalizedString -Key "Could not get the primary storage key for storage account '{0}'. Unable to apply any diagnostics extensions." -ArgumentList "$storageAccount")
         }
     }
     
     return $diagnosticsConfigurations
 }
 
-Write-Host "Entering script Publish-AzureCloudDeployment.ps1"
-
-import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
+Write-Verbose "Entering script Publish-AzureCloudDeployment.ps1"
 
-Write-Host "ConnectedServiceName= $ConnectedServiceName "
-Write-Host "ServiceName= $ServiceName"
-Write-Host "ServiceLocation= $ServiceLocation"
-Write-Host "StorageAccount= $StorageAccount"
-Write-Host "CsPkg= $CsPkg"
-Write-Host "CsCfg= $CsCfg"
-Write-Host "Slot= $Slot"
-Write-Host "AllowUpgrade= $AllowUpgrade"
+Write-Verbose "ConnectedServiceName= $ConnectedServiceName "
```

Comment:
I would prefer that these remain Write-Host.  Deploying Azure solutions can take considerable time and it will speed up debugging if we left these as -Host.  (I've gone back and forth on this for awhile but am convinced Host is the way to go for these solutions.)  Otherwise, you need to re-run a long process in order to debug what the values are.